File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -663,10 +667,156 @@ using (FileStream fileStreamInput = new FileStream(@"Template.pptx", FileMode.Op
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PPTX-to-Image-conversion/Apply-custom-fallback-fonts).
+### Modify the exiting fallback fonts
+
+The following code example demonstrates how user can modify or customize the existing fallback fonts using *FontNames* API while converting a PowerPoint presentation to Image.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Load the PowerPoint presentation into stream.
+using (FileStream fileStreamInput = new FileStream(@"Template.pptx", FileMode.Open, FileAccess.Read))
+{
+    //Open the existing PowerPoint presentation with loaded stream
+    using (IPresentation pptxDoc = Presentation.Open(fileStreamInput))
+    {
+        //Initialize the PresentationRenderer to perform image conversion
+        pptxDoc.PresentationRenderer = new PresentationRenderer();
+        //Use a sets of default FallbackFont collection to IPresentation
+        pptxDoc.FontSettings.FallbackFonts.InitializeDefault();
+        // Customize a default fallback font name
+        // Modify the Hebrew script default font name as "David"
+        pptxDoc.FontSettings.FallbackFonts[5].FontNames = "David";
+        //Convert PowerPoint slide to image as stream
+        using (Stream stream = pptxDoc.Slides[0].ConvertToImage(ExportImageFormat.Jpeg))
+        {
+            //Reset the stream position
+            stream.Position = 0;
+            //Create the output image file stream
+            using (FileStream fileStreamOutput = File.Create("Output.jpg"))
+            {
+                //Copy the converted image stream into created output stream
+                stream.CopyTo(fileStreamOutput);
+            }
+        }
+    }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+### Supported script types
+
+The following table illustrates the supported script types by .NET PowerPoint library (Presentation) in Presentation to Image(Portable) conversion.
```

Comment:
The following table illustrates the supported script types by the .NET PowerPoint library (Presentation) in Presentation to Image(Portable) conversion.

Reply:
Modified the sentences